src/Input/Silk.NET.Input.Common/Interfaces/ICursor.cs

```diff
@@ -0,0 +1,48 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+namespace Silk.NET.Input.Common
+{
+    /// <summary>
+    /// An interface representing a mouse cursor.
+    /// </summary>
+    public interface ICursor
+    {
+        /// <summary>
+        /// Hotspot on the X axis.
+        /// </summary>
+        int HotspotX { get; set; }

```

Comment:
Is this not just `IMouse.X`?

Reply:
X and Y would be the cursor's position on-screen, not the hotspot, which is where the click actually happens.

Reply:
Is the hotspot relative to the cursor position, e.g. (0,4)?

Reply:
Yeah